docs/course_materials/exercises/test_exercises/README_run_tests.md

```diff
@@ -0,0 +1,22 @@
+# Checking Exercises
+
+## Running the checks
+
+First, make sure you are using python 3.7 or later.  
+
+        `pipenv --python /usr/local/bin/python3.7 shell`
+
+Second, change directory to `ROSE/docs/course_materials/exercises/test_exercises` before running the tests.
```

Comment:
We are already in ROSE directory. so cd should be to docs/course_materials/exercises/test_exercises